frontend/components/auction/debt/DebtAuction.vue

```diff
@@ -0,0 +1,259 @@
+<template>
+    <TextBlock :title="`Debt auction #${auctionId}`">
+        <div class="my-3">
+            <Alert v-if="auctionError && auctionError.showBanner" :message="auctionError.error" type="error" />
+            <Alert
+                v-if="auction && auction.state === 'ready-for-collection'"
+                message="This auction has ended and is ready for collection"
+                type="info"
+            />
+        </div>
+        <div v-if="auction">
+            <div v-if="requiresRestart">
+                <AuctionRestartPanel
+                    :wallet-address="walletAddress"
+                    :is-explanations-shown="isExplanationsShown"
+                    :is-restarting="auctionActionState === 'restarting'"
+                    :is-connecting="isConnecting"
+                    auction-type="surplus"
```

Comment:
Do we need to add a new auction type here or do we combine the auction types into `compensation`? A naming schema first brought up by @KirillDogadin-std in #398.